src/vm/exceptionhandling.cpp

```diff
@@ -1930,12 +1930,12 @@ CLRUnwindStatus ExceptionTracker::ProcessOSExceptionNotification(
             // DispatcherContext->EstablisherFrame's value
             // represents the CallerSP of the current frame.
             UINT_PTR EnclosingClauseCallerSP = (UINT_PTR)pDispatcherContext->EstablisherFrame;
-#elif defined(_TARGET_AMD64_)            
-            // Extract the CallerSP from RegDisplay on AMD64
+#elif defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
```

Comment:
I am not sure if we want to use the ARM or AMD64 way for x86 - did we already make a choice in this matter? I 
If we follow the amd64 way, then we also need to fix the code here:
https://github.com/dotnet/coreclr/blob/master/src/vm/exceptionhandling.cpp#L2843-L2849
and here:
https://github.com/dotnet/coreclr/blob/master/src/vm/exceptionhandling.cpp#L3109-L3115
(and if not, we should modify the comments at those places to mention x86).